### DISCUSSION TITLE
How to install the package with poetry add ?

### DISCUSSION THREAD

**Discussion by HuangChiEn** [python-poetry/poetry#7627] [Q&A]
I have check the document guidence. However, simply `poetry add $PKG_NAME` does not work. 

At the first time, i have try to `poetry add torch` without going to the poetry shell, and the lock file also reveal. Since the installing procedure is too fast, i wonder the package not even be installed! And not surprised, `poetry show` show noting.  

To display convenience, i going to the poetry shell, and simply show that poetry add not work at all for installing opencv-python. How could i fix this issue ?
![image](https://user-images.githubusercontent.com/52521165/223633220-f75151ac-3c4c-44bf-9318-3b7474fa49f5.png)

FAQ : 
1. In colab, i have simply  `pip install poetry` 
2.  i have try to `poetry add` without poetry shell
3. `poetry show` shows nothing.
4. below the project folder, i have saw the pyproject.toml and the lock file after conduct `poetry init` and `poetry add` respectively.
5. poetry == 1.4.0
6. poetry env info 
![image](https://user-images.githubusercontent.com/52521165/223634196-1936decc-2cbf-4059-92ee-6ee9ae41b857.png)

Any feedback will be appreciated!!

**Comment by ralbertazzi** [comment]
I don't know what's the behavior on Colab, but perhaps Poetry created its own virtual env and you haven't activated it?

Anyways, I would suggest you to open a proper issue on Poetry, please make sure to provide all requirements requested in the issue template as to ensure reproducibility.
